internal/test/data/hdf5/hdf5.go

```diff
@@ -0,0 +1,235 @@
+//
+// Copyright (C) 2019-2022 vdaas.org vald team <vald@vdaas.org>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Package hdf5 is load hdf5 file
+package hdf5
+
+import (
+	"os"
+	"reflect"
+	"strconv"
+
+	"gonum.org/v1/hdf5"
+
+	"github.com/vdaas/vald/internal/errors"
+	"github.com/vdaas/vald/internal/io"
+	"github.com/vdaas/vald/internal/net/http/client"
+)
+
+type Data interface {
+	Download() error
+	Read() error
+	GetName() DatasetName
+	GetPath() string
+	GetTrain() [][]float32
+	GetTest() [][]float32
+	GetNeighbors() [][]int
+}
+
+type DatasetName int
+
+const (
+	FASHION_MNIST_784_EUC DatasetName = iota
```

Comment:
it's looks like Java style please use go style const variable
```suggestion
	FashionMNIST784Euclidean DatasetName = iota
```